packages/language/src/workspace/lifecycle.ts

```diff
@@ -80,19 +79,17 @@ export async function tokenize(
 }
 
 export function parse(compilationUnit: CompilationUnit): Program {
-  PliParserInstance.input = compilationUnit.tokens;
-  const ast = PliParserInstance.parse();
-  compilationUnit.ast = ast;
-  compilationUnit.diagnostics.addAll(
-    DiagnosticCategory.Parser,
-    parserErrorsToDiagnostics(PliParserInstance.errors),
-  );
+  const debugging = process.env.NODE_ENV === "development";
```

Comment:
Suggestion: Can we move this into a utility somewhere? This way, we can access the info directly from the parser and don't need to pass it into the parser. And also make this a bit more web-proof by doing something like this:


```suggestion
  const debugging = typeof process !== 'undefined' && process.env?.NODE_ENV === "development";
```